test/dashdb.injection.test.js

```diff
@@ -0,0 +1,44 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-dashdb
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+/* eslint-env node, mocha */
+process.env.NODE_ENV = 'test';
+
+require('./init.js');
+require('should');
+
+var db, Customer;
+
+describe('parameterized SQL', function() {
+  before(function(done) {
+    db = global.getDataSource();
+    Customer = db.define('Customer', {
+      name: {type: String},
+      pass: {type: String},
+    });
+    db.automigrate('Customer', function(err) {
+      if (err) return done(err);
+      var samples = [
+          {name: 'Zoe', pass: '123'},
+          {name: 'John', pass: '456'},
+      ];
+      Customer.create(samples, done);
+    });
+  });
+  it('does not return result with SQL injection', function(done) {
+    // generate injectedSQL as `SELECT * FROM Cusotmer WHERE name ="" or ""=""`
```

Comment:
lol thanks, actually I updated it on local and about to squash commits then push it with dependency change(if needed.) together.